command/meta.go

```diff
@@ -75,10 +72,10 @@ type ApiMeta struct {
 func (m *ApiMeta) GlobalFlagSet(cmd string) *flag.FlagSet {
 	f := flag.NewFlagSet(cmd, flag.ContinueOnError)
 
-	f.StringVar(&m.gateEndpoint, "gate-endpoint", "http://localhost:8084",
+	f.StringVar(&m.Config.GateEndpoint, "gate-endpoint", "http://localhost:8084",
```

Comment:
Not sure this is evaluated in the right order, I would expect `--gate-endpoint` to override what is specified in the config.